orchestration/nersc.py

```diff
@@ -0,0 +1,147 @@
+from authlib.integrations.requests_client import OAuth2Session
+from authlib.oauth2.rfc7523 import PrivateKeyJWT
+import time, json, os, logging
+from orchestration.ptycho_jobscript import *
```

Comment:
If we had time, I'd like to change nersc.py  so that it did not depend on anything in ptycho, and could easily be used for other beamlines.